src/main/java/careconnect/ui/PersonDetailCard.java

```diff
@@ -64,17 +58,6 @@ public PersonDetailCard(Person person, MainWindow.FocusItemUpdater focusItemUpda
         phone.setText(person.getPhone().value);
         address.setText(person.getAddress().value);
         appointmentDate.setText(person.getAppointmentDate().toString());
-        addressLink.setOnMouseClicked(e -> {
-            try {
-                String q = person.getAddress().value.replaceAll(" ", "+");
-                URI link = new URI("https://www.google.com/maps/place/" + q);
-                Desktop.getDesktop().browse(link);
-            } catch (URISyntaxException | IOException ex) {
-                // TODO: handle this properly. we should show a ui to inform the user that
```

Comment:
This should be removed to avoid violating the no internet constraint, thanks MJ!